CHANGELOG.md

```diff
@@ -1,10 +1,31 @@
 # Changelog
 
+## 3.1.0 - 2022-02-28
+### Added
+- Accept & decline invitations from web
+- Conflict calendars for appointments
+- Limit how far in the future appointments can be booked
+### Changed
+- Time-insensitive background jobs are now run at off-peak times
```

Comment:
(Nextcloud 23+)

Reply:
Fair enough. @miaulalala maybe you can include that when you do the v3.1 final releaes PR :)